### DISCUSSION TITLE
How do I disable or make it really hard to throw windows?

### DISCUSSION THREAD

**Discussion by Piste** [rxhanson/RectanglePro-Community#508] [Q&A] [answered]
I currently have an issue with Rectangle Pro whereby if I press ⌥ and move my cursor, the window gets thrown. I have tried to disable this by removing the modifier or adding all the modifiers that I could but no matter what, pressing ⌥ still throws the window. What am I doing wrong? Am I looking at the wrong settings?
![image](https://github.com/rxhanson/RectanglePro-Community/assets/5099194/8f37d17c-0d62-4580-8fab-967fc02cf8e6)

**Comment by rxhanson** [comment, marked as answer]
This is the "Quick Throw". You can uncheck the "Enabled" checkbox in the Quick Throw section, or remove the ones that have the option key configured.

I'm going to remove the option key ones from the default configuration, since those are pretty easy to accidentally trigger.

**Comment by Piste** [comment]
Ah, thank you very much!!
